package.json

```diff
@@ -174,9 +174,9 @@
     "eslint-plugin-prettier": "2.6.1",
     "eslint-plugin-promise": "3.6.0",
     "eslint-plugin-react": "7.9.1",
-    "extract-text-webpack-plugin": "3.0.2",
+    "extract-text-webpack-plugin": "4.0.0-beta.0",
     "flow-bin": "0.80.0",
-    "html-webpack-plugin": "2.30.1",
+    "html-webpack-plugin": "^3.2.0",
```

Comment:
please use exact versioning

Reply:
when was the last time we went through and updated the modules?

Reply:
Depends on which dependencies you are referring to but this PR represents an upgrade of all our webpack related modules to the latest and greatest...